test/parallel/test-crypto-authenticated-stream.js

```diff
@@ -9,6 +9,9 @@ const path = require('path');
 const stream = require('stream');
 const tmpdir = require('../common/tmpdir');
 
+if (!common.hasCrypto)
```

Comment:
move this block right up the top just below `const common...`, your error is coming from `require('crypto')` for builds that aren't compiled with crypto so you need to bail before that.